src/main/java/core/basesyntax/service/WriterToCsv.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface WriterToCsv {
```

Comment:
The interface name WriterToCsv suggests that it's specifically for writing to CSV files, which might be a form of hardcoding. Consider a more generic name like `WriterService` unless the interface is strictly for CSV outputs.